src/views/RoomDetail.vue

```diff
@@ -49,17 +64,92 @@ export default {
   created () {
     this.roomID = this.$route.params.roomID
   },
-  mounted () {
-    const db = this.$firebase.firestore()
-    db
-      .collection('Rooms')
-      .doc(this.roomID)
-      .get()
-      .then(doc => {
-        this.room = doc.data()
-      })
+  async mounted () {
+    await this.getRoomKeyValuePairs()
+    await this.getRoomCheckIns()
+    await this.getRoomCheckInsTomorrow()
+    await this.fetchUserIDList()
+    await this.isUserSignedIn()
+    await this.isUserSignedInTomorrow()
+    await this.roomCapacityToday()
+    await this.roomCapacityTomorrow()
   },
   methods: {
+    getRoomKeyValuePairs: async function () {
+      await $db()
+        .collection('Rooms')
+        .doc(this.roomID)
+        .get()
+        .then(doc => {
+          this.room = doc.data()
+          this.room.checkIns = []
+          this.room.checkInsTomorrow = []
+        })
+    },
+    getRoomCheckIns: async function () {
+      await $db()
+        .collection('Rooms')
+        .doc(this.roomID)
+        .collection('CheckIn')
+        .where('date', '==', this.currentDate)
+        .get()
+        .then(querySnapshot => {
+          querySnapshot.forEach((checkIn) => {
+            this.room.checkIns = checkIn.data()
+          })
+        })
+    },
+    getRoomCheckInsTomorrow: async function () {
+      await $db()
+        .collection('Rooms')
+        .doc(this.roomID)
+        .collection('CheckIn')
```

Comment:
Hier bitte auch zusammenfassen wie oben.

Reply:
Hier genau so.

Reply:
In Teams geklärt. 👍